src/stock-card-summary-list/stock-card-summary-list.html

```diff
@@ -85,6 +85,6 @@ <h2 ng-hide="vm.facility && vm.program">{{'stockCardSummaryList.stockOnHand' | m
             </tr>
         </tbody>
     </table>
-    <openlmis-pagination ng-if="!vm.offline()" list="vm.displayStockCardSummaries" paged-list="vm.pagedList" pagination-id="'stockCardSummaries'"/>
+    <openlmis-pagination ng-if="!vm.offline()" list="vm.displayStockCardSummaries" paged-list="vm.pagedList" pagination-id="'stockCardList'"/>
```

Comment:
should it be in scope of this PR ?

Reply:
It is not related, however it's an old bug which should have been corrected some time ago - ticket for it is not created, so I thought that we can resolve this bug now

Reply:
I can delete this line if needed